axum-extra/src/extract/multipart.rs

```diff
@@ -118,7 +115,22 @@ impl Multipart {
             .map_err(MultipartError::from_multer)?;
 
         if let Some(field) = field {
-            Ok(Some(Field { inner: field }))
+            // multer still uses http 0.2 which means we cannot directly expose
+            // `multer::Field::headers`. Instead we have to eagerly convert the headers into http
+            // 1.0
+            //
+            // When the next major version of multer is published we can remove this.
+            let mut headers = HeaderMap::with_capacity(field.headers().len());
+            headers.extend(field.headers().clone().into_iter().map(|(name, value)| {
```

Comment:
No need to create a new `HeaderMap` and then immediately turn it into a destructuring iterator, right?

```suggestion
            headers.extend(field.headers().iter().cloned().map(|(name, value)| {
```